docs/configuration.md

```diff
@@ -106,6 +106,10 @@ max_batch:
 enforce_post:
   # Enable the feature
   enabled: true
+
+log:
+  format: text #text or json
+  log_graphql_errors: false #enable/disables logging of graphql errors
```

Comment:
I would not put this option here, this is logging configuration, and shouldnt be mixed with a functional rule

Reply:
changed